app/views/devise/registrations/new.html.erb

```diff
@@ -6,6 +6,11 @@
     <%= devise_error_messages! %>
 
     <div class="field">
+      <%= f.label :username %><br />
+      <%= f.username_field :name, autocomplete: "off" %>
```

Comment:
There is no such method `username_field` on the form object, so that's why you're seeing an error here - you probably want `f.text_field` (also this is not SimpleForm, it's just the regular Rails form helpers, so `f.input` won't work either)